test/common/utxobased/keymanager/coins/bitcointransactiontest.spec.ts

```diff
@@ -203,6 +204,128 @@ describe('bitcoin transaction creation and signing test', function () {
+    /*
+      This here is the rawtransaction as assembled below:
+      0200000001f9f34e95b9d5c8abcd20fc5bd4a825d1517be62f0f775e5f36da944d9452e550000000006b483045022100c86e9a111afc90f64b4904bd609e9eaed80d48ca17c162b1aca0a788ac3526f002207bb79b60d4fc6526329bf18a77135dc5660209e761da46e1c2f1152ec013215801210211755115eabf846720f5cb18f248666fec631e5e1e66009ce3710ceea5b1ad13ffffffff01905f0100000000001976a9148bbc95d2709c71607c60ee3f097c1217482f518d88ac00000000
+      The test case deconstructs the value, script pubkey and locktime values to show some deserialized values.
+      This deserialization is not required in the usual form from the caller.
+      It is enough to pass the full previous rawtransaction.
+    */
+    const { psbtBase64 } = makeTx({
+      forceUseUtxo: [],
+      coin: 'bitcoin',
+      currencyCode: 'BTC',
+      setRBF: false,
+      freshChangeAddress: '1KRMKfeZcmosxALVYESdPNez1AP1mEtywp',
+      feeRate: 0,
+      subtractFee: false,
+      utxos: [
+        {
+          id: '0',
+          scriptType: ScriptTypeEnum.p2pkh,
+          txid:
+            '7d067b4a697a09d2c3cff7d4d9506c9955e93bff41bf82d439da7d030382bc3e',
+          // prev_tx only for non segwit inputs
+          scriptPubkey,
+          value: '80000',
+          blockHeight: 0,
+          spent: false,
+          script:
+            '0200000001f9f34e95b9d5c8abcd20fc5bd4a825d1517be62f0f775e5f36da944d9' +
+            '452e550000000006b483045022100c86e9a111afc90f64b4904bd609e9eaed80d48' +
+            'ca17c162b1aca0a788ac3526f002207bb79b60d4fc6526329bf18a77135dc566020' +
+            '9e761da46e1c2f1152ec013215801210211755115eabf846720f5cb18f248666fec' +
+            '631e5e1e66009ce3710ceea5b1ad13ffffffff01' +
+            // value in satoshis (Int64LE) = 0x015f90 = 90000
+            '905f010000000000' +
+            // scriptPubkey length
+            '19' +
+            // scriptPubkey
+            scriptPubkey +
+            // locktime
+            '00000000',
+          vout: 0
+        }
+      ],
+      targets: [
+        { address: segwitAddress, value: 300 },
+        { address: segwitAddress, value: 200 },
+        { address: segwitAddress, value: 100 }
+      ],
+      outputSort: 'targets'
+    })
+    const signedTx = await signTx({
+      psbtBase64,
+      privateKeys: [privateKey],
+      coin: 'bitcoin'
+    })
+    expect(signedTx.hex).to.equal(
+      '02000000013ebc8203037dda39d482bf41ff3be955996c50d9d4f7cfc3d2097a694a7b067d000000006a47304402203d4b4d8f7be9219914e822684ce72001367081101654aaeeaa734e6aa934972e022072154039d101819e8c78fc3a11132e5a46a3538546db68c8a7818128b02de88101210365db9da3f8a260078a7e8f8b708a1161468fb2323ffda5ec16b261ec1056f455ffffffff042c010000000000001600148bbc95d2709c71607c60ee3f097c1217482f518dc8000000000000001600148bbc95d2709c71607c60ee3f097c1217482f518d64000000000000001600148bbc95d2709c71607c60ee3f097c1217482f518d28360100000000001976a914ca0d36044e0dc08a22724efa6f6a07b0ec4c79aa88ac00000000'
```

Comment:
We should really cleanup these tests to expect some more readable output fixture data. How are we to know this output hex data is correct? I only request writing up a breakaway task to fix this.

Reply:
I actually decoded the output value using https://live.blockcypher.com/btc/decodetx/ and verified the order of the outputs